oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LucenePropertyIndex.java

```diff
@@ -1477,6 +1478,14 @@ public boolean visit(FullTextTerm term) {
 
             private boolean visitTerm(String propertyName, String text, String boost, boolean not) {
                 String p = getLuceneFieldName(propertyName, pr);
+                // below condition is for contains(*, 'xyz')
+                if (propertyName == null) {
+                    // Lucene indexes don't support inference, so we should remove queryInferenceConfig
```

Comment:
Should we rather write - don't support vector queries - it's easier to understand.

Also an example of queryInferenceConfig that is getting removed in comment would be helpful.

Reply:
Updated